contracts/test/CompoundGovernorSetQuorum.t.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.26;
+
+import {ProposalTest} from "contracts/test/helpers/ProposalTest.sol";
+
+contract CompoundGovernorSetQuorumTest is ProposalTest {
+    function _submitPassAndExecuteProposalToSetNewQuorum(address _proposer, uint256 _amount) public {
```

Comment:
nit: rather than these 2 internal methods, i might prefer one internal method that generates the quorum proposal, and then call the "submit and pass" or "submit and fail" down in the test body itself. what do you think?

Reply:
I had actually moved to that as I was looking into another issue.. will put that code in here.

Reply:
Done: 5e4a6397fa3c15fead8e83223d42d6e88e6f81c1